eslint.config.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I suggest moving this content to `.eslintrc.json` and removing this file as a duplicate. The other file has the correct name but the incorrect content

Reply:
Done! moved the content from eslint.config.json to .eslintrc.json and them deleted the eslint.config.json file